### PR TITLE
[v0.25] Updates use of db seed to prisma seed

### PR DESCRIPTION
**Pull request by dthyresson**
Docs, cookbook refer to `yarn rw db seed`, but with 0.25 this has been replaced with `yarn rw prisma seed`.

### CODE REVIEW

File: TUTORIAL2.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ git clone https://github.com/redwoodjs/redwood-tutorial
 cd redwood-tutorial
 yarn install
 yarn rw db up
-yarn rw db seed
+yarn rw prisma db seed
```
Comment (thedavidprice) [inline TUTORIAL2.md:47]: @dthyresson My assumption is that Rob is going to do a full pass through both Tutorials. So maybe check with him before we do this. I'd hate to create merge conflicts on his end.

Make sense?

Comment (cannikin) [reply TUTORIAL2.md:47]: Yeah I'm on Part 1 right now but am gonna go through Part 2 next. If you want to close this one I'll be sure to cover it in mine!

**Review by thedavidprice — APPROVED** [review]

**Comment by cannikin** [comment]
And we did this seeds.js -> seed.js rename in CRA right?

**Comment by thedavidprice** [comment]
@cannikin 
> And we did this seeds.js -> seed.js rename in CRA right?

Yes, I did. Also will include instructions to change in the Code Mod section of Release Notes.

**Comment by thedavidprice** [comment]
Closing in favor of #556 edits

**Pull request closed without merging**